### PR TITLE
Post/decentralized verifiable credentials

### PR DESCRIPTION
**Pull request by gkontos**
Post about Verifiable Credentials and Decentralized Personal Datastores

### CODE REVIEW

File: posts/decentralized-data-storage-finance.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Use Cases
+To date, distributed stores have seen a few use cases implemented within finance. The primary use cases implemented thus far fall into two categories.  1) customer convenience and 2) personalized customer experience.  Both these categories of use case are enabled by the concept of interoperability.  [explain concept]. Presumably this same interoperability is what would enable what  And the same interoperability that will enable "(improving their [the customer] experiences access the digital context)[https://canada-ca.github.io/PCTF-CCP/docs/RelatedPolicies/WEF_Digital_Identity.pdf]" as described in a World Economic Forum paper from Janurary 2020.
```
Comment (dsharpe) [inline posts/decentralized-data-storage-finance.md:27]: "Presumably this same interoperability is what would enable what" <- This sentence is incomplete

File: posts/decentralized-data-storage-finance.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Use Cases
+To date, distributed stores have seen a few use cases implemented within finance. The primary use cases implemented thus far fall into two categories.  1) customer convenience and 2) personalized customer experience.  Both these categories of use case are enabled by the concept of interoperability.  [explain concept]. Presumably this same interoperability is what would enable what  And the same interoperability that will enable "(improving their [the customer] experiences access the digital context)[https://canada-ca.github.io/PCTF-CCP/docs/RelatedPolicies/WEF_Digital_Identity.pdf]" as described in a World Economic Forum paper from Janurary 2020. 
+ 
+in 2020 the Solid Platform was used by the NatWest Bank to provide more a personalized experiences for customers [(source: Inrupt.com](https://inrupt.com/solid-enterprise-natwest-bbc). The NatWest integration
```
Comment (dsharpe) [inline posts/decentralized-data-storage-finance.md:29]:
```suggestion
in 2020 the Solid Platform was used by the NatWest Bank to provide more personalized experiences for customers [(source: Inrupt.com](https://inrupt.com/solid-enterprise-natwest-bbc). The NatWest integration 
```

File: posts/decentralized-data-storage-finance.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+[Mozilla, the EFF, and the ACLU of California](https://blog.mozilla.org/netpolicy/2020/08/06/by-embracing-blockchain-a-california-bill-takes-the-wrong-step-forward/) in 2020 listed concerns over the use of the W3C Verified Credentials in a pilot program to communicate COVID-19 test results and 'other medical information'.   
+
+## Conclusion
+
+The personal data store and verified credentials ecosystem is just beginning to take it's first steps.  With the direction of the W3C specifications to promote interoperability there are a few open projects and companies which are gaining the capability to address the issues of data use transparecy, data security, and customer experience.  At the moment VC and personal data stores are not ready for wide scale adoption, but worth familiarizing with, investigating use cases, and monitoring the growth of implementations.  These technologies appear to be a solution which will continue to develop over the next 2-4 years and provide opportunities for banks to reduce duplication, improve their customer's experience, and also improve data privacy.
```
Comment (dsharpe) [inline posts/decentralized-data-storage-finance.md:51]:
```suggestion
The personal data store and verified credentials ecosystem is just beginning to take it's first steps.  With the direction of the W3C specifications to promote interoperability there are a few open projects and companies which are gaining the capability to address the issues of data use transparecy, data security, and customer experience.  At the moment VC and personal data stores are not ready for wide scale adoption, but worth familiarizing with, investigating use cases and monitoring the growth of implementations.  These technologies appear to be a solution which will continue to develop over the next 2-4 years and provide opportunities for banks to reduce duplication, improve their customer's experience, and also improve data privacy.
```

**Review by dsharpe — CHANGES_REQUESTED** [review]

File: posts/decentralized-data-storage-finance.md (hunk 3 above)
Comment (gkontos) [reply posts/decentralized-data-storage-finance.md:51]: Not a fan of the oxford comma?

File: posts/decentralized-data-storage-finance.md (hunk 1 above)
Comment (gkontos) [reply posts/decentralized-data-storage-finance.md]: fixed

**Review by dsharpe — APPROVED** [review]

**Pull request merged**